detox/src/artifacts/utils/ArtifactPathBuilder.test.js

```diff
@@ -73,6 +73,20 @@ describe(ArtifactPathBuilder, () => {
 
       expect(asPosixPath(artifactPath)).toBe('/tmp/subdir/✗ test/1.log');
     });
+
+    it('should append (2) invocations count for a test that fails for the second time', () => {
+      const testSummary = {title: '', fullName: 'test', status: 'failed', invocations: 2 };
+      const artifactPath = pathBuilder.buildPathForTestArtifact('1.log', testSummary);
+
+      expect(asPosixPath(artifactPath)).toBe('/tmp/subdir/✗ test (2)/1.log');
+    });
+
+    it('should append (3) invocations count for a test that passes for the third time', () => {
+      const testSummary = {title: '', fullName: 'test', status: 'passed', invocations: 3 };
+      const artifactPath = pathBuilder.buildPathForTestArtifact('1.log', testSummary);
+
+      expect(asPosixPath(artifactPath)).toBe('/tmp/subdir/✓ test (3)/1.log');
+    });
```

Comment:
Just wondering: isn't this test redundant?...

Reply:
I adhere to the blackbox approach, @d4vidi.
In my opinion, there were two ways to implement this.
Either we omit ` (${invocations})` suffix, when the status is `passed` or we add it nevertheless.
Therefore, I **fixate** the behavior and confirm that I as an author intend it to be this way.

Reply:
Actually, I see now that I missed the difference in `status` value. You can ignore my comment.